### PR TITLE
Added Chrysanthemum

### PR DESCRIPTION
**Pull request by Mysc**
Was tempted to re-add my Ripened glutinous rice tea. It's different from the normal glutinous rice tea I brought before I swear.

### CODE REVIEW

File: MEETINGS.md [hunk 1]
```diff
@@ -344,11 +345,14 @@
 [Yuyupas]: http://www.yuyupas.com
 
 <!-- Name -->
+<<<<<<< HEAD
+=======
```
Comment (rkazakov) [inline MEETINGS.md]: You have a conflict here ^^^. Remove this.

File: MEETINGS.md [hunk 2]
```diff
@@ -344,11 +345,14 @@
...
 [Jeju Canola & Honey]: https://www.osulloc.com/kr/en/shop/item/blended/5663
 [Advent Christmas Tea]: https://gewurzhaus.com.au/product/advent-christmas-tea
 [Breakfast Ceylon (OPX)]: http://adoretea.com.au/New-Tea/breakfast-ceylon-opx.html
 [Mulled Wine Magic]: http://www.t2tea.com/en/uk/tea/limited-edition/mulled-wine-magic-loose-leaf-feature-tin-T140AI119.html
 [Orange & Cinnamon]: https://www.woolworths.com.au/shop/productdetails/791152/twinings-orange-cinnamon-tea
+>>>>>>> 114e12ecadc692d62d39584228ee1d9d31830c4c
```
Comment (rkazakov) [inline MEETINGS.md]: And here ^^^

**Review by rkazakov — CHANGES_REQUESTED** [review]
Request for changes.

**Comment by Mysc** [comment]
removed conflicts

**Review by rkazakov — APPROVED** [review]

**Pull request merged**